### PR TITLE
Fighting The Winter's Chill: A Cosmetic Expansion

### PR DESCRIPTION
**Pull request by CosmicPhoenix**
### Fighting The Winter's Chill: A Cosmetic Expansion
Coats

## About The Pull Request

Adds in new custom winter coats lovingly hand-sprited by Papaporo Paprito for all the staff heads. Also tweaked item placement in Head lockers so custom items are not at the bottom.

## Why It's Good For The Game

Allows the Heads of Staff to still look distinct during the winter season aboard Space Station 13. No longer will the Heads be running around in a sea of similar winter coats, you'll now be able to pick them out of the crowd.

## Changelog
:cl:
add: Custom winter coats hand tailored by moths have been issued to all Heads of Staff. ~NT
/:cl:

### CODE REVIEW

**Review by CosmicPhoenix — COMMENTED** [review]
Doesn't edit jumpsuits so I apparently didn't break them :wee:

File: code/FulpstationCode/wintercoats.dm [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	mob_overlay_icon = 'icons/fulpicons/phoenix_nest/wintercoats.dmi'
+	icon = 'icons/fulpicons/phoenix_nest/wintercoathoods.dmi'
+	icon_state = "winterhood_hop"
+
+/obj/item/clothing/suit/hooded/wintercoat/cargo/head
+	name = "\improper Quarter Master's cargo winter coat"
+	desc = "It's just a regular cargo winter coat... it seems the Quarter Master still isn't a head of staff..."
+
+/* Commented out for now. This'll go into the QM's locker on Halloween or something.
+/obj/item/clothing/suit/cardborg/qm
```
Comment (CosmicPhoenix) [inline code/FulpstationCode/wintercoats.dm]: Just ignore this. This isn't the start of a backdoor to take over Fulp or anything.

**Comment by TheSwain** [comment]
fastest merge of all time - thanks for keeping locker additions ABOVE the last existing entry in the locker <3

**Comment by Paprito** [comment]
happy moth noises

**Pull request merged**